mapfishapp/src/main/webapp/app/addons/goto/js/main.js

```diff
@@ -25,12 +25,12 @@ GEOR.Addons.Goto = Ext.extend(GEOR.Addons.Base, {
         this.store = new Ext.data.JsonStore({
             data: this.options.projections,
             idProperty: "srs",
-            fields: ["srs", "name"]
+            fields: ["srs", "name", {name: "decimalPrecision", defaultValue: 2}]
```

Comment:
Note that I put 2 to preserve the default value for minimal changes in current production environments.

But maybe we could change it to 6.